src/lib/cache/subCache.cpp

```diff
@@ -77,6 +77,8 @@ EntityInfo::EntityInfo(const std::string& _entityId, const std::string& _entityT
 
   if (isPattern)
   {
+    // FIXME P5: recomp error should be captured? have a look to other usages of regcomp()
+    // in order to see how it works
```

Comment:
Doesn't sound like a bad idea.
Capture error and return error response ...

http://linux.die.net/man/3/regcomp

We should create an issue about it